exporter/otelarrowexporter/README.md

```diff
@@ -244,44 +244,107 @@ exporters:
 
 ### Batching Configuration
 
-This exporter includes a new, experimental `batcher` configuration for
-batching in the `exporterhelper` module, but this mode is disabled by
-default.  This batching support works when combined with
-`queue_sender` functionality.
+### Option 1: Batching with back-pressure
 
+To configure an OpenTelemetry Collector pipeline for both batching and
+back-pressure, use of a custom component, the Concurrent Batch Processor,
+available in the OTel-Arrow project repository, is required. We have not
+included this in the Collector-Contrib repository because equivalent
+functionality is being added as a standard exporter-batcher mechanism and the
+new exporter-batcher functionality is still experimental.
+
+When the [Concurrent Batch Processor](https://github.com/open-telemetry/otel-arrow/blob/main/collector/processor/concurrentbatchprocessor/README.md) is configured, parallel batches of data are
+exported with no limit on concurrency. This configuration requires that
+receivers apply memory limits or admission control. While this is our preferred
+configuration, it is just one of several reasonable setups. As an example
+configuration:
+
+```yaml
+exporters:
+  otelarrow:
+    # place gRPC, otel-arrow, retry, and timeout settings here
+    batcher:
+	  enabled: false
+    sending_queue:
+      enabled: false
+receivers:
+  otelarrow:
+    # otelarrow supports OTLP and OTel-Arrow with admission control
+    admission:
+      request_limit_mib: 128
+processors:
+  concurrentbatch:
+service:
+  pipelines:
+    traces:
+      exporters: [otelarrow]
+      processors: [concurrentbatch]
+      receivers: [otelarrow]
 ```
+
+### Option 2: Batching with a persistent queue
+
+The OpenTelemetry Collector has a built-in persistent queue mechanism which
+supplies back-pressure corresponding with disk write speed. In this mode,
```

Comment:
Hmm can you elaborate more on "back-pressure corresponding with disk write speed"?  i.e. the queue_sender in general does not provide back-pressure because both the bounded_queue and the persistent_queue it use the sizedChannel implementation -- i.e. if channel is full we just return an error https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/internal/queue/sized_channel.go#L60

Reply:
It seems you are pointing out a different kind of back pressure related to disk size and write speed -- is it possible for the persistent queue to keep up with the flow of traffic but the sizedChannel does not keep up,  leading to drops? i.e. do we still ultimately have the same back-pressure issue related to the sizedChannel for both the bounded_queue and persistent_queue?

Reply:
Thanks @moh-osman3. In other words, "it's worse than I thought".  I'm surprised and saddened to learn that even w/ a peristent store enabled, we can't have back-pressure. This is all the more reason to fix exporterhelper. I'm ok leaving a slightly inaccurate document -- and surprised there's no option to block on the disk.

Reply:
@moh-osman3 thank you for raising this point. I reviewed the code and you are correct, there's no back-pressure here in case "channel len can be out of sync with used size". I'm not sure what to make of this--it sounds like a bug if the persistent queue returns queue-full when the disk-based queue is not full.

```
	// for persistent queue implementation, channel len can be out of sync with used size. Attempt to put it
	// into the channel. If it is full, simply returns ErrQueueIsFull error. This prevents potential deadlock issues.
```

I'm willing to let this documentation stand and encourage users who encounter this to file a bug against the persistent queue.  Anyway, I agree. The persistent queue should wait for context timeout, not take a default branch and immediately return queue full!